src/logging.c

```diff
@@ -31,6 +35,13 @@ void rcutils_logging_initialize()
   if (!g_rcutils_logging_initialized) {
     g_rcutils_logging_output_handler = &rcutils_logging_console_output_handler;
     g_rcutils_logging_severity_threshold = RCUTILS_LOG_SEVERITY_INFO;
+    // Check for the environment variable to disable output formatting
+    const char * output_format;
+    const char * ret;
+    ret = rcutils_get_env("RCUTILS_CONSOLE_OUTPUT_FORMAT", &output_format);
+    if (!ret && strcmp(output_format, "") != 0) {
+      g_rcutils_logging_output_format_string = output_format;
```

Comment:
According to the docblock of `rcutils_get_env` this won't work:

> The c-string which is returned in the env_value output parameter is only valid until the next time this function is called, because it is a direct pointer to the static storage.

So this needs to copy the string.

Reply:
I've tried to do this dynamically in 1162ac4a0681e46a14a006aebaafd9666d1f758c but FWIU I'll need to deallocate it myself, and I don't see anywhere appropriate to do that since there isn't a fini for the logging module. So I'm going to switch this to a large static allocation that will truncate.